### ISSUE TITLE
[feat]: Right Click on Table - TRUNCATE

### ISSUE THREAD

**Issue by millerjp** [axonops/axonops-workbench#542]
Do after #539 

To truncate a table in Apache Cassandra, you use the `TRUNCATE` command. This command removes all data from the specified table immediately and irreversibly, as well as any materialized views derived from that table.

## Requirements
Users right click on a table, select the truncate table option, we generate the CQL, users validates it and then we run it and then we execute the command against the table.

Important difference is to use `CONSISTENCY ALL;` - include this statment in the generated CQL.

Before executing the TRUNCATE display a warning dialog saying

---

### Warning: Truncate Operation

**This operation is **irreversible** and will permanently delete all data in the table.**

- By executing this command, **all rows** in the specified table will be removed **immediately**. 
- **Materialized views** derived from this table will also be truncated.
- **Snapshots:** If your Cassandra cluster is configured to take snapshots before truncation, a snapshot of the table's current data will be created for recovery purposes. Remember to remove this snapshot if they are not needed.
  - **Clean up snapshots**: Use `nodetool clearsnapshot` to remove snapshots if they are no longer needed.
  - **Disable automatic snapshots**: Modify `cassandra.yaml` to disable snapshots if you do not want this behavior.

**Please confirm that you want to proceed with this operation, understanding its irreversible nature and potential impact on snapshots.**

---

## Syntax

The syntax for the `TRUNCATE` command in Cassandra is:

```sql
TRUNCATE [ TABLE ] [<keyspace_name>.]<table_name>;
```

- `keyspace_name` is optional if the table is in the current keyspace.
- `table_name` is the name of the table you want to truncate.

## Examples

Here are some examples of how to use the `TRUNCATE` command:

```sql
-- Truncate a table in the current keyspace
TRUNCATE users;

-- Truncate a table in a specific keyspace Use this CQL
TRUNCATE keyspace_name.users;

-- Truncate a table with the optional TABLE keyword. 
TRUNCATE TABLE products; 
```

**Comment by millerjp** [comment]
Done
